include/util/FitsComparator.h

```diff
@@ -0,0 +1,36 @@
+//
```

Comment:
We need license statement at the top of the file. Copy from one of the files and change your name in the License.